developerportal/templates/admin/login.html

```diff
@@ -0,0 +1,42 @@
+{% extends "admin/login.html" %}
+{% load i18n app_tags %}
+
+{% block content %}
+
+  {% use_conventional_auth as conventional_auth_enabled %}
+
+  {% if conventional_auth_enabled %}
+
+    {{block.super}}
+
+  {% else %}
+
```

Comment:
I wanted to keep the error messages visible, just in case, but the Django Admin base template didn't really have a great block-inheritance pattern to lean on.